charts/kube-prometheus-stack/values.yaml

```diff
@@ -993,7 +995,7 @@ kubeControllerManager:
     ## Enable scraping kube-controller-manager over https.
     ## Requires proper certs (not self-signed) and delegated authentication/authorization checks
     ##
-    https: false
+    https: null
```

Comment:
Can you add a line above that says something like:
"If null or unset, determined dynamically based on target Kubernetes version

Reply:
Sure.